spec/index.html

```diff
@@ -715,8 +719,11 @@ <h4 id="query"><a class="toc-anchor" href="#query" >§</a> Query</h4>
 <li>The message object <strong><strong>MUST</strong></strong> contain a <code>descriptor</code> property, and its value <strong><strong>MUST</strong></strong> be a JSON object composed as follows:
 <ul>
 <li>The object <strong><strong>MUST</strong></strong> contain a <code>method</code> property, and its value <strong><strong>MUST</strong></strong> be the string <code>CollectionsQuery</code>.</li>
+<li>The object <strong><strong>MAY</strong></strong> contain a <code>filter</code> property, and if present its value <strong><strong>MUST</strong></strong> be an object that <strong><strong>MAY</strong></strong> contain the following properties:
```

Comment:
What is the point of `<strong><strong>`? It seems to me that these doubles should be undoubled throughout...